Issues running dynamodb as a config rule

**Describe the bug**
Running a simple tag filter on dynamodb does not work when running as a config-rule, while  the same policies run locally or in lambda periodic do work

**To Reproduce**
Steps to reproduce the behavior:
Have some dynamodb table in an account, apply policy.  Failure manifests in the created lambdas logs


**Expected behavior**
Policy to run and either find a match or none


**Background (please complete the following information):**

 - Custodian Version: 0.9.10

 - Cloud Provider: aws
 - Policy: [please exclude any account/sensitive information]
```yaml
policies:
- name: dynamodb-tag-config-compliance
  resource: aws.dynamodb-table
  mode:
    type: config-rule
    role: custodian
  filters:
  - or:
    - "tag:Name": absent
    - "tag:Contact": absent
    - "tag:Environment": absent

```
 - Traceback: [if applicable, please exclude sensitive/account information]


```
[ERROR] TypeError: unsupported operand type(s) for /: 'datetime.datetime' and 'float'
Traceback (most recent call last):
  File "/var/task/custodian_policy.py", line 4, in run
    return handler.dispatch_event(event, context)
  File "/var/task/c7n/handler.py", line 165, in dispatch_event
    p.push(event, context)
  File "/var/task/c7n/policy.py", line 1140, in push
    return mode.run(event, lambda_ctx)
  File "/var/task/c7n/policy.py", line 853, in run
    resources = super(ConfigRuleMode, self).run(event, lambda_context)
  File "/var/task/c7n/policy.py", line 437, in run
    resources = self.resolve_resources(event)
  File "/var/task/c7n/policy.py", line 835, in resolve_resources
    return [source.load_resource(self.cfg_event['configurationItem'])]
  File "/var/task/c7n/resources/dynamodb.py", line 23, in load_resource
    resource['CreationDateTime'] = datetime.fromtimestamp(resource['CreationDateTime'] / 1000.0)
```

--- a/c7n/resources/dynamodb.py
+++ b/c7n/resources/dynamodb.py
@@ -2,7 +2,6 @@
 # SPDX-License-Identifier: Apache-2.0
 from botocore.exceptions import ClientError
 from concurrent.futures import as_completed
-from datetime import datetime
 
 from c7n.actions import BaseAction, ModifyVpcSecurityGroupsAction
 from c7n.filters.kms import KmsRelatedFilter

--- a/c7n/resources/dynamodb.py
+++ b/c7n/resources/dynamodb.py
@@ -20,13 +20,6 @@ class ConfigTable(query.ConfigSource):
 
     def load_resource(self, item):
         resource = super(ConfigTable, self).load_resource(item)
-        resource['CreationDateTime'] = datetime.fromtimestamp(resource['CreationDateTime'] / 1000.0)
-        if ('BillingModeSummary' in resource and
-                'LastUpdateToPayPerRequestDateTime' in resource['BillingModeSummary']):
-            resource['BillingModeSummary'][
-                'LastUpdateToPayPerRequestDateTime'] = datetime.fromtimestamp(
-                    resource['BillingModeSummary']['LastUpdateToPayPerRequestDateTime'] / 1000.0)
-
         sse_info = resource.pop('Ssedescription', None)
         if sse_info is None:
             return resource

--- a/tools/c7n_salactus/requirements.txt
+++ b/tools/c7n_salactus/requirements.txt
@@ -10,7 +10,7 @@ functools32==3.2.3-2
 futures==3.3.0
 ipaddress>=1.0.17
 itsdangerous==1.1.0
-Jinja2==2.10.3
+Jinja2==2.11.3
 jmespath>=0.9.0
 jsonschema>=2.5.1
 MarkupSafe==1.1.1

